src/picongpu/include/plugins/radiation/debug/PIConGPUVerboseLogRadiation.hpp

```diff
@@ -38,6 +38,7 @@ DEFINE_VERBOSE_CLASS(PIConGPUVerboseRadiation)
     DEFINE_LOGLVL(2,SIMULATION_STATE);
```

Comment:
yes, there is absolutely no reason to use just an other log level here.

we should refactor this in a follow up pull request (post `0.1` release)